README.md

```diff
@@ -124,11 +124,19 @@ If you want to load config files from a directory other than the project root yo
 ```ts
 import { loadConfig } from 'config-dug';
 
-loadConfig('config');
+loadConfig('config', { files: true, environment: false, secrets: false });
 ```
 
 This will import your config files from the `config` directory. The path you specify must be relative to your project root.
 
+The second parameter contains options to specify where to load configs from. By default, Config Dug will load from all possible sources.
```

Comment:
`By default, Config Dug will load from all possible sources`
I wonder if this is something we should be cautious now that we'll use config-dug in FE projects. Maybe we should  not load all by default OR split config-dug into a BE and FE module. I dunno, just guessing.